### DISCUSSION TITLE
Regarding Skywalking server Hardware requirement

### DISCUSSION THREAD

**Discussion by mkannance** [apache/skywalking#11842] [Q&A]
Hi all, 

I want to configure apache skywalking server for my java application monitoring, My question is how much hardware needed for install initial apache skywaking server and if i added multiple agent then how much occupied in server for every agent node. kindly any one help in this, thanks in advance.

**Comment by wu-sheng** [comment]
If you asked for Java agent performance benchmark, there is an old one, https://skyapmtest.github.io/Agent-Benchmarks/. But the kernel is generally as same as the latest 9.1.

About the server node, I don't know. It depends on how much payload you are going to run, how many spans are you going to collect, how much extra information you plan to collect.

**Reply by mkannance** [reply]
thank you for your information.
